Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -56,4 +56,8 @@
 
 + (instancetype)wmf_utcShortDayNameShortMonthNameDayOfMonthNumberDateFormatter;
 
++ (instancetype)wmf_yearMonthDayZDateFormatter;
```

Comment:
These are set to use 'en' locale id... just double-checking as the formatters above don't seem to set this.